src/compas_viewer/components/dialog.py

```diff
@@ -44,18 +44,17 @@ class CameraSettingsDialog(QDialog, Base):
 
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
         super().__init__()
         self.setWindowTitle("Camera Settings")
 
         # Create layout for spin boxes
         self.layout = QVBoxLayout(self)
         self.spin_boxes = {}
         current_camera = self.viewer.renderer.camera
-
         coordinates = {
-            "Camera_Target": [("X", current_camera.target.x, 0, 10000), ("Y", current_camera.target.y, 0, 10000), ("Z", current_camera.target.z, 0, 10000)],
-            "Camera_Position": [("X", current_camera.position.x, 1, 10000), ("Y", current_camera.position.y, 1, 10000), ("Z", current_camera.position.z, 1, 10000)],
+            "Camera_Target": [("X", current_camera.target.x, -10000, 10000), ("Y", current_camera.target.y, -10000, 10000), ("Z", current_camera.target.z, -10000, 10000)],
+            "Camera_Position": [("X", current_camera.position.x, -10000, 10000), ("Y", current_camera.position.y, -10000, 10000), ("Z", current_camera.position.z, -10000, 10000)],
```

Comment:
Is there way to set min and max as infinite? Other wise these numbers needs to be much larger / smaller like 10^10.

Reply:
Also try always use kwargs only when initing classes